README.md

```diff
@@ -92,7 +92,7 @@ jobs:
         run: "spin --version"
 ```
 
-## Push Spin app to a Registry - `fermyon/actions/spin/push@v1`
+## Push Spin app to an OCI Registry - `fermyon/actions/spin/push@v1`
```

Comment:
As we've been generally running down our use of "OCI" (using it more when we need to expand on "what is a registry"), I'd consider keeping the original title and changing the link in the front matter instead.  What do you reckon?

Reply:
Sounds good, thanks @itowlson